static/panes/flags-view.js

```diff
@@ -1,212 +0,0 @@
-// Copyright (c) 2021, Tom Ritter
```

Comment:
Not sure how this works with copyright, cc @mattgodbolt

Reply:
For all the ts conversions I've done I've left the copyright as-is if it's not "Compiler Explorer Authors." This dilemma is very reminiscent of the ship of Theseus :P Additive definitely seems the safe way to go, I can update.